app/controllers/stories/tagged_articles_controller.rb

```diff
@@ -0,0 +1,72 @@
+module Stories
+  class TaggedArticlesController < ApplicationController
+    before_action :set_cache_control_headers, only: :index
+
+    SIGNED_OUT_RECORD_COUNT = 60
```

Comment:
non-blocking: I wonder if it's a good idea here for this class instead rely on `StoriesController::SIGNED_OUT_RECORD_COUNT` instead of re-declaring it here. That way we could keep that variable consistent

Reply:
How about we extract the constant to `Settings`? Maybe `Settings::UserExperience`?

Reply:
@kuei0221 constants in Settings are mostly meant to be modified by admins in the Admin panel, if this is "just a constant" it's probably overkill. We can still port it over separately as I think it should be a "product decision", in case configuring that count is actually needed